### PR TITLE
Localise toolbar home button and ruleset description tooltips

### PR DESCRIPTION
**Pull request by Domilz**
Interpreted issue [14083](https://github.com/ppy/osu/issues/14083) as one could work on any part of the localisation and create a pull request. Not sure if it is okay to use the localisation in the Toolbarbutton files. I have made some sv.resx files of the respective parts in osu-resources which I will create a pull request for. [Osu-resources pr](https://github.com/ppy/osu-resources/pull/217).

### CODE REVIEW

**Comment by peppy** [comment]
I'd probably move all the new localisable strings into `ToolbarStrings`, as each individual button / area is only going to have a couple each. Makes more sense to group.

https://github.com/ppy/osu/blob/5d8bd1de288efacf2481aa5f30f2bb2abdddedc4/osu.Game/Localisation/ToolbarStrings.cs#L8-L7

File: osu.Game/Localisation/ToolbarStrings.cs [hunk 1]
```diff
@@ -19,6 +19,22 @@ public static class ToolbarStrings
...
+        /// <summary>
+        /// "return to the main menu"
+        /// </summary>
+        public static LocalisableString HomeHeaderDescription => new TranslatableString(getKey(@"header_description"), @"return to the main menu");
+
+        /// <summary>
+        /// "play some"
+        /// </summary>
+
+        public static LocalisableString RulesetHeaderDescription => new TranslatableString(getKey(@"header_description"), @"play some");
```
Comment (peppy) [inline osu.Game/Localisation/ToolbarStrings.cs:36]: This needs to take an argument, as depending on language the ruleset's name could move around in the sentence.

If you're using rider or visual studio, you should be able to use the bundled tool to create this:

JetBrains Rider-EAP 2022-10-12 at 04 54 29

to create something like:

```csharp
        /// <summary>
        /// "play some {0}"
        /// </summary>
        public static LocalisableString PlaySome(string arg0) => new TranslatableString(getKey(@"play_some"), @"play some {0}", arg0);
```

**Review by peppy — CHANGES_REQUESTED** [review]

**Comment by Joehuu** [comment]
I'm going to take over this PR to ease PR count, but also because localisation is being added everywhere right now.

**Comment by peppy** [comment]
Is this ready for review?

**Comment by Joehuu** [comment]
Yeah, should be simple enough anyway.

**Review by peppy — APPROVED** [review]

**Pull request merged**